From 7cf319e79002681ff7914a84bbc1e9af93d93e0f Mon Sep 17 00:00:00 2001
From: Eric Brown <browne@vmware.com>
Date: Fri, 15 May 2015 14:54:16 -0700
Subject: Typo in openstack client help

The openstack client output for the help of --os-user-id states
"longin" instead of "login".  The openstack client gets it's help
output from the keystoneclient.

Change-Id: I7c92a82cd60b2835d98101200cf641b46dd145b4
Closes-Bug: #1455673
---
 keystoneclient/auth/identity/v2.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/keystoneclient/auth/identity/v2.py b/keystoneclient/auth/identity/v2.py
index 8eaa9c5..c25ddfd 100644
--- a/keystoneclient/auth/identity/v2.py
+++ b/keystoneclient/auth/identity/v2.py
@@ -153,7 +153,7 @@ class Password(Auth):
                        dest='username',
                        deprecated_name='username',
                        help='Username to login with'),
-            cfg.StrOpt('user-id', help='User ID to longin with'),
+            cfg.StrOpt('user-id', help='User ID to login with'),
             cfg.StrOpt('password', secret=True, help='Password to use'),
         ])
 
-- 
cgit v0.11.2

